packages/reports/app/styles/navi-reports/components/report-view.less

```diff
@@ -105,3 +105,8 @@
         justify-content: flex-end;
     }
 }
+
+.number-format-dropdown__container {
```

Comment:
Why is this here? Shouldn't it be here: https://github.com/yahoo/navi/blob/a236f1798cc0c0f6750443995ad01bf901100b19/packages/visualizations/app/styles/navi-visualizations/components/number-format-dropdown.less

Reply:
The dropdown is displaying correctly in visualization dummy app, but not in report dummy app, so I guess we should just apply this style to report

Reply:
This feels out of place in the `report-view`

Reply:
Yeah, It is kind of tricky since dropdown is rendered in a wormhole

Reply:
work for me